docs/sdkx_android/delegates.mdx

```diff
@@ -127,6 +133,62 @@ Helpshift.setHelpshiftEventsListener(new HelpshiftEventsListener() {
   }
 ```
 
+### User Click On Action Event {#user-click-on-action}
+
+This event is triggered when the user clicks on the link or call action of an action card message.
+
+- Event Name : `HelpshiftEvent.ACTION_CLICKED`
+- Event data: 
+  - `HelpshiftEvent.DATA_ACTION `
+  - `Helpshift.DATA_ACTION_TYPE `
+  - `Helpshift.DATA_ACTION_TYPE_CALL `
+  - `Helpshift.DATA_ACTION_TYPE_LINK `
+
+| Key (Constant)                | Key (Raw)  | Type   |
+|-------------------------------|------------|--------|
+| HelpshiftEvent.ACTION_CLICKED  | userClickOnAction | String |
+| HelpshiftEvent.DATA_ACTION  | actionType | String |
+| Helpshift.DATA_ACTION_TYPE  | actionData | String |
+| Helpshift.DATA_ACTION_TYPE_CALL  | call | String |
+| Helpshift.DATA_ACTION_TYPE_LINK  | link | String |
+
+<Admonition type="info" title="Note">
+
+The key constants are available from SDK X 10.3.0 onwards. For older SDK X versions, please use the raw keys instead.
+
+</Admonition>
+
+```java
+Helpshift.setHelpshiftEventsListener(new HelpshiftEventsListener() {
+  @Override
+  public void onEventOccurred(@NonNull String eventName, Map<String, Object> data) {
+    switch(eventName){
+      case HelpshiftEvent.ACTION_CLICKED:
+          // With Key Constants
+          String actionType = (String) data.get(HelpshiftEvent.DATA_ACTION_TYPE);
+          String actionData = (String) data.get(HelpshiftEvent.DATA_ACTION);
+
+          -------------------- OR --------------------
+
+          // With Key Raw Values
+          String actionType = (String) data.get("actionType");
+          String actionData = (String) data.get("actionData");
+
+          // `Utils.isEmpty` is null and empty Check
+          if (Utils.isEmpty(actionType) || Utils.isEmpty(actionData)) {
+            Log.d(TAG, "Event Received for " + eventName + " with actionType or action Data as empty");
+            return;
+          }
+
+          if (actionType.equals(HelpshiftEvent.DATA_ACTION_TYPE_LINK)) {
+            Log.d(TAG, "Event Received for " + eventName + " action type " + actionType + " link " + actionData);
+          } else if (actionType.equals(HelpshiftEvent.DATA_ACTION_TYPE_CALL)) {
+            Log.d(TAG, "Event Received for " + eventName + " action type " + actionType + " number " + actionData);
```

Comment:
same log statement for both if-else part.
either merge if-else to if condition to remove actionType from log statement and mention the actual action itself.